README.md

```diff
@@ -50,9 +54,12 @@ putasset(token, {
     owner: 'coderaiser',
     repo: 'putasset',
     tag: 'v1.0.0',
-    filename: 'realease.zip'
+    filename: 'realease.zip',
+    force: true
 }).catch((error) => {
     console.error(error.message);
+}).then((url) => {
+    console.log(`Upload success, download url: ${url}`);
```

Comment:
I think we can output this information only when `--loud` flag present.

Reply:
It is what I first though to do but I'd rather leave it like this as the CLI tool will often be used in a bash script. 
If we only output the url this:
````bash
URL=$(putasset-r putasset -o coderaiser -t v1.0.0 -f "release.zip")
````
Can be done, if on the other hand we only print the URL if --loud is used there is some parsing to do to get only the last line.